src/states/common/approved.rs

```diff
@@ -59,6 +59,7 @@ pub trait Approved: Relocated {
     fn handle_neighbour_merge_event(&mut self) -> Result<(), RoutingError>;
 
     /// Handles an accumulated `SectionInfo` event.
+    /// If triggered during initial, an ack message shall not be sent out.
```

Comment:
Could you rephrase the comment? I'm not sure what "triggered during initial" means.